zerver/management/commands/backup.py

```diff
@@ -46,7 +46,7 @@ def handle(self, *args: Any, **options: Any) -> None:
 
             with open(os.path.join(tmp, "zulip-backup", "os-version"), "w") as f:
                 print(
-                    "{DISTRIB_ID} {DISTRIB_CODENAME}".format(**parse_lsb_release()),
+                    "{NAME} {VERSION_CODENAME}".format(**parse_lsb_release()),
```

Comment:
I think the function called would need to be `parse_os_release()` here. I haven't read the code but the commit message implies this.

Reply:
I forgot to do that because I wanted to introduce minimal change that works. Added in the second commit.